src/components/ReanimatedDrawerLayout.tsx

```diff
@@ -0,0 +1,745 @@
+        props.hideStatusBar,
+        props.onDrawerClose,
+        props.onDrawerOpen,
+        props.statusBarAnimation,
+        drawerWidth,
+        sideCorrection,
+        isDrawerShowing,
+      ]
+    );
+
+    const handleRelease = React.useCallback(
+      ({
+        nativeEvent,
+      }: {
+        nativeEvent: {
+          translationX: number;
+          velocityX: number;
+          x: number;
+        };
+      }) => {
+        'worklet';
+        let { translationX: dragX, velocityX, x: touchX } = nativeEvent;
+
+        if (drawerPosition !== positions.Left) {
+          // See description in _updateAnimatedEvent about why events are flipped
+          // for right-side drawer
+          dragX = -dragX;
+          touchX = containerWidth - touchX;
+          velocityX = -velocityX;
+        }
+
+        const gestureStartX = touchX - dragX;
+        let dragOffsetBasedOnStart = 0;
+
+        if (drawerType === 'front') {
+          dragOffsetBasedOnStart =
+            gestureStartX > drawerWidth ? gestureStartX - drawerWidth : 0;
+        }
+
+        const startOffsetX =
+          dragX +
+          dragOffsetBasedOnStart +
+          (isDrawerOpen.value ? drawerWidth : 0);
+
+        const projOffsetX = startOffsetX + DRAG_TOSS * velocityX;
+
+        const shouldOpen = projOffsetX > drawerWidth / 2;
+
+        if (shouldOpen) {
+          animateDrawer(drawerWidth, velocityX);
+        } else {
+          animateDrawer(0, velocityX);
+        }
+      },
+      [
+        animateDrawer,
+        containerWidth,
+        drawerPosition,
+        drawerType,
+        drawerWidth,
+        isDrawerOpen.value,
+      ]
+    );
+
+    const openDrawer = (options: DrawerMovementOption = {}) => {
+      'worklet';
+      animateDrawer(
+        drawerWidth,
+        options.velocity ? options.velocity : 0,
+        options.speed
+      );
+    };
+
+    const closeDrawer = React.useCallback(
+      (options: DrawerMovementOption = {}) => {
+        'worklet';
+        animateDrawer(
+          0,
+          options.velocity ? options.velocity : 0,
+          options.speed
+        );
+      },
+      [animateDrawer]
+    );
+
+    const overlayDismissGesture = React.useMemo(
+      () =>
+        Gesture.Tap()
+          .maxDistance(25)
+          .onEnd(() => {
+            if (isDrawerOpen.value && props.drawerLockMode !== 'locked-open') {
+              closeDrawer();
+            }
+          }),
+      [closeDrawer, isDrawerOpen.value, props.drawerLockMode]
+    );
+
+    const overlayAnimatedStyle = useAnimatedStyle(() => ({
+      opacity: openValue.value,
+      backgroundColor: props.overlayColor ?? defaultProps.overlayColor,
+    }));
+
+    const renderOverlay = React.useCallback(() => {
+      return (
+        <GestureDetector gesture={overlayDismissGesture}>
+          <Animated.View
+            pointerEvents={isDrawerOpen.value ? 'auto' : 'none'}
+            animatedProps={overlayAnimatedProps}
```

Comment:
This will always overwrite `pointerEvents` above so one of those isn't needed (since `isDrawerOpen` is a shared value it should go through `animatedProps`)

Reply:
fixed in [eaba51f](https://github.com/software-mansion/react-native-gesture-handler/pull/3146/commits/eaba51ff0e7590e653a0026e50397675d81af4e1)